.azure-pipelines/generation-pipeline.yml

```diff
@@ -485,6 +490,32 @@ stages:
           parameters:
             repoName: 'msgraph-beta-typescript-typings'
 
+- stage: stage_typescript_sdk_beta
+  dependsOn:
+  - stage_build_and_publish_kiota
+  - stage_beta_openapi
+  condition: |
+    and
+    (
+      eq(dependencies.stage_build_and_publish_kiota.result, 'Succeeded'),
+      eq(dependencies.stage_beta_openapi.result, 'Succeeded')
+    )
+  jobs:
+  - job: typescript_sdk_beta
+    steps:
+    - template: generation-templates/language-generation-kiota.yml
+      parameters:
+        language: 'typescript'
+        version: 'beta'
+        repoName: 'msgraph-sdk-typescript'
```

Comment:
could we target the beta repo instead or use the v1 version to avoid mismatches?